source/help/getting-started/signing-in.rst

```diff
@@ -0,0 +1,126 @@
+Signing in
+==========
+
+--------------
+
+This documentation refers to Mattermost v3.0+. For v2.2, see `archived
+documentation`_.
+
+You can sign in to your team from the root page of your Mattermost site
+``https://domain.com/``. After signing in, you will be able to view a
+list of your teams and select which one to open.
+
+Sign In Methods
+---------------
+
+There are several options for signing in to your team depending on how
+your System Admin has configured your server.
+
+Email Address or Username Sign In
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When enabled by your System Admin, you can sign in with the username or
+email address used to create your account.
+
+If you have forgotten your password, you can reset it by clicking **I
+forgot my password** on the sign in screen, or contact your System Admin
+for help resetting your password.
+
+GitLab Single-Sign-On (SSO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When enabled by your System Admin, you can sign in using your GitLab
+account using a one-click sign in option.
+
+Google Single-Sign-On (`Enterprise Edition`_)
```

Comment:
See the system console documentation for the new way we are marking EE features in the docs